### PR TITLE
Create jobs board layout

### LINKED ISSUE #73: Create jobs page layout

### PR DESCRIPTION
**Pull request by FatumAbdullahi**
close #73

### CODE REVIEW

File: src/Components/Dashboard/Cards/MarketingCard.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React, {Component} from "react";
```
Comment (ahmedhamodi) [inline src/Components/Dashboard/Cards/MarketingCard.js:1]: Why are you creating a new marketing card? Use the job application card

File: src/Components/Dashboard/Jobs.js [hunk 2]
```diff
@@ -1,25 +1,351 @@
 import React, {Component} from "react";
 import {makeStyles} from "@material-ui/core/styles";
+import Grid from "@material-ui/core/Grid";
+import MarketingCard from "./Cards/MarketingCard";
```
Comment (ahmedhamodi) [inline src/Components/Dashboard/Jobs.js:4]: Use job application card.

File: src/Components/Dashboard/Cards/MarketingCard.js (hunk 1 above)
Comment (FatumAbdullahi) [reply src/Components/Dashboard/Cards/MarketingCard.js:1]: Sorry didn't see the Job application Card. fixed it now tho.

File: src/Components/Dashboard/Jobs.js (hunk 2 above)
Comment (FatumAbdullahi) [reply src/Components/Dashboard/Jobs.js:4]: fixed

**Pull request merged**